CHANGELOG.md

```diff
@@ -64,11 +64,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added: Crossway - Easier boost only method for item
 - Changed: Tower of Light - Reduced gravityless SJ slope jump to tower chamber to Beginner
 - Fixed: The Door in Tower of Light Access that leads to Ruined Shrine is now a normal Door instead of a Wave Beam Door.
+- Changed: Ruined Nursery Bombless Standables Logic Adjustments
+- Added: Ruined Nursery Bombless w/ Boost strat
+- Added: Training Chamber Ghost Skip (Door to Training Chamber Access -> Pickup (Energy Tank))
```

Comment:
We don't have to be so specific in the changelog to specify each direction of a bi-directional connection change.